homeassistant/components/tado/weather.py

```diff
@@ -0,0 +1,107 @@
+"""Support for the Tado weather service."""
+from homeassistant.components.weather import WeatherEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import TEMP_CELSIUS
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import (
+    ATTRIBUTION,
+    CONDITIONS_MAP,
+    DATA,
+    DOMAIN,
+    SIGNAL_TADO_UPDATE_RECEIVED,
+)
+from .entity import TadoHomeEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+):
+    """Set up the Tado weather platform."""
+
+    tado = hass.data[DOMAIN][entry.entry_id][DATA]
+
+    async_add_entities([TadoWeatherEntity(tado)], True)
```

Comment:
```suggestion
    async_add_entities([TadoWeatherEntity(tado)])
```

Does this need to update on add?